fastlane/Fastfile.swift

```diff
@@ -68,4 +68,12 @@ class Fastfile: LaneFile {
         Symbol.downloadFromAppStore(versionNumber: versionNumber, buildNumber: buildNumber)
         Symbol.uploadAppStoreToCrashlytics(versionNumber: versionNumber, buildNumber: buildNumber)
     }
+
+    func buidAndUploadToTestFlightLane() {
```

Comment:
```suggestion
    func buildAndUploadToTestFlightLane() {
```

Reply:
It seemed I lost my mind at that time, lol. Good catch anyway! Fixed in [98e46f7](https://github.com/nimblehq/ios-templates/pull/364/commits/98e46f7e3cc9ce7a08c03c5a98d7fb5b19e48b83)